### PR TITLE
Create group feature

### PR DESCRIPTION
**Pull request by Bearypop**
Add create group feature.

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedGroup.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    public JsonAdaptedGroup(@JsonProperty("number") int number,
+                            @JsonProperty("members") List<JsonAdaptedPerson> members) {
+        this.number = number;
+        if (members != null) {
+            this.members.addAll(members);
+        }
+    }
+
+    /**
+     * Converts a given {@code SocialMedia} into this class for Jackson use.
```
Comment (PearlynnT) [inline src/main/java/seedu/address/storage/JsonAdaptedGroup.java:39]:
```suggestion
     * Converts a given {@code Group} into this class for Jackson use.
```

File: src/main/java/seedu/address/storage/JsonAdaptedGroup.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public JsonAdaptedGroup(Group source) {
+        number = source.getNumber();
+        members.addAll(source.getMembers().stream()
+                .map(JsonAdaptedPerson::new)
+                .collect(Collectors.toList()));
+    }
+
+    /**
+     * Converts this Jackson-friendly adapted social media object into the model's {@code SocialMedia} object.
```
Comment (PearlynnT) [inline src/main/java/seedu/address/storage/JsonAdaptedGroup.java:49]:
```suggestion
     * Converts this Jackson-friendly adapted group object into the model's {@code Group} object.
```

File: src/main/java/seedu/address/storage/JsonAdaptedGroup.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     *
+     * @throws IllegalValueException if there were any data constraints violated in the adapted social media.
```
Comment (PearlynnT) [inline src/main/java/seedu/address/storage/JsonAdaptedGroup.java:51]:
```suggestion
     * @throws IllegalValueException if there were any data constraints violated in the adapted group.
```

File: src/main/java/seedu/address/model/group/Group.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+     * @param number A valid group number.
+     */
+    public Group(int number) {
+        this.number = number;
+    }
+
+    /**
+     * Constructs a {@code Group}.
+     *
+     * @param number
```
Comment (PearlynnT) [inline src/main/java/seedu/address/model/group/Group.java:31]: Provide parameter description

File: src/main/java/seedu/address/model/group/Group.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+     * @param members
```
Comment (PearlynnT) [inline src/main/java/seedu/address/model/group/Group.java:32]: Provide parameter description

**Review by PearlynnT — COMMENTED** [review]
Edit javadocs

File: src/main/java/seedu/address/storage/JsonAdaptedGroup.java (hunk 1 above)
Comment (Bearypop) [reply src/main/java/seedu/address/storage/JsonAdaptedGroup.java:39]: Oops

**Pull request merged**